src/ign.cc

```diff
@@ -36,6 +36,14 @@ char* g_argv[] =
 //////////////////////////////////////////////////
 extern "C" IGNITION_GUI_VISIBLE char *ignitionVersion()
 {
+  std::string home;
+  ignition::common::env(IGN_HOMEDIR, home);
+
+  std::string recordPathMod = ignition::common::joinPaths(home,
+        ".ignition", "gazebo", "log",ignition::common::timeToIso(IGN_SYSTEM_TIME()));
```

Comment:
Ignition GUI is used by other applications besides Gazebo, so we should save this to `~/.ignition/gui` by default.

```suggestion
        ".ignition", "gui", "log",ignition::common::timeToIso(IGN_SYSTEM_TIME()));
```

For `ign-gazebo`, we can add the same logic to its `ign.cc`. It shouldn't conflict with this one because the `ign gazebo` command doesn't directly call the `ign gui` command.

Reply:
Oh ok. So there's going to be gui logs in both `~/.ignition/gui` and `~/.ignition/gazebo` ?

Reply:
When users run `ign gui`, the logs will end up in `~/.ignition/gui`, when they run `ign gazebo`, they should go into `~/.ignition/gazebo`.

Reply:
Done in 
b9d8345